scripts/PearlPowderISIS/pearl_calib_factory.py

```diff
@@ -0,0 +1,411 @@
+# pylint: disable=too-many-branches,too-many-statements, superfluous-parens
+
+from mantid.simpleapi import *
+
+
+def get_calibration_dir(cycle, tt_mode, pearl_file_dir):
+    if (cycle == "15_4"):
+        instver = "new2"
+        calfile = pearl_file_dir + "Calibration\\pearl_offset_15_3.cal"
```

Comment:
@FedeMPouzols This is where the system test is failing.. Should I change them all to forward slash?